Bremtown Case Study

SondraE opened the pull request:
## Related Issue(s)
[#844](https://github.com/oddbird/oddleventy/issues/844)


## Steps to test/reproduce
Work list
Bremtown detail page

jgerigmeyer commented on content/work/bremtown.md:
```diff
@@ -0,0 +1,95 @@
+  a searchable product web application
+  to showcase their cabinetry products
+  to dealers, designers, and firms.
+  We also helped redesign their
+  brand, logo, structure,
+  and narrative of their marketing site
```
```suggestion
  We also helped redesign their
  brand and logo, and the structure
  and narrative of their marketing site
```

jgerigmeyer commented on content/work/bremtown.md:
```diff
@@ -0,0 +1,95 @@
+---
+title: Bremtown Cabinetry
+sub: Spec Book web app, Marketing website & CMS
```
The capitalization here seems a bit odd.

jgerigmeyer commented on content/work/bremtown.md:
```diff
@@ -0,0 +1,95 @@
+  and increased lead generation.
+---
+
+{% import 'quotes.macros.njk' as quotes %}
```
```suggestion
```

SondraE commented:
@jgerigmeyer I don't see the modified images showing up, but I did remove that little tab, and updated that other items from your review.

SondraE commented:
@jgerigmeyer 

> I think we still need a quote (and pic) from Tim.

You want to ask him about those or shall I?

jgerigmeyer commented:
> @jgerigmeyer
> 
> > I think we still need a quote (and pic) from Tim.
> 
> You want to ask him about those or shall I?

@SondraE I can -- would you be willing to draft the questions for him and then I'll email it?

SondraE commented:
@jgerigmeyer I replaced the quote section with the Bremtown image. I realized I hadn't included links to Bremtown.com or the spec book, so I added those as well. I also added "Nonprofit Sector" tag to Bremtown, Metecho, MetaDeploy, Emmy's Workshop, and Phamaly. If this looks good, then I'll merge it.
